Wiki/text-notes.md

```diff
@@ -1,107 +1,94 @@
-# Text-notes
-Trilium uses awesome [CKEditor 5](https://ckeditor.com/ckeditor-5/) as its editing component.
+# Text Notes
 
-Formatting
-----------
+Trilium utilizes the powerful [CKEditor 5](https://ckeditor.com/ckeditor-5/) as its text editing component.
 
-Trilium text note interface doesn't show any toolbars or formatting options by default, these needs to be brought up by:
+## Formatting Options
 
-1.  selecting text will bring up an inline toolbar:
+The Trilium text note interface does not display toolbars or formatting options by default. These can be accessed by:
 
-![](api/images/voFa4y9vMkRi/text-notes-formatting-inline.p)
+%%{WARNING}%% Image does not exist in repo
+![](api/images/aQT4C1G1rjUk/text-notes-formatting-inline.p)
 
-1.  clicking on the block toolbar:
+1. Selecting text to bring up an inline toolbar.
 
+%%{WARNING}%% Image does not exist in repo
 ![](api/images/aQT4C1G1rjUk/text-notes-formatting-block.pn)
 
-Read only vs. editing mode
---------------------------
+2. Clicking on the block toolbar.
 
-Text notes are normally opened in edit mode, however there are two cases when they are open in read-only mode:
 
-*   they are long and thus would take time to load so by default we open them as read only which is much quicker
-*   or the note has `readOnly` [label](attributes.md)
+## Read-Only vs. Editing Mode
 
-In both cases, it is possible to switch to editable mode again.
+Text notes are usually opened in edit mode. However, they may open in read-only mode under the following circumstances:
 
-General Formatting
-------------------
+- The note is long and would take time to load, so it is opened in read-only mode by default for quicker access.
+- The note has a `readOnly` [label](attributes.md).
 
-Trilium uses the CKEditor, so any formatting that the CKEditor supports should be available in Trilium. For example:
+In both cases, it is possible to switch back to editable mode.
 
-**Bold** – Type `**text**` or `__text__`
+## General Formatting
 
-_Italic_ – Type `*text*` or `_text_`
+Since Trilium uses CKEditor, any formatting supported by CKEditor is available. Examples include:
```

Comment:
Since Trilium uses CKEditor, all of its formatting options are available here. You may use the graphical toolbar shown above, or enter formatting such as markdown markdown directly in the text. Examples include: